src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java

```diff
@@ -77,5 +77,9 @@ public void beforeAndAfter() {
         Map<Class<?>, Integer> a = new HashMap<Class<?>, Integer>();
         Map<Class<?>, Integer> b = (Map<Class<?>, Integer>) a;
     }
+    Object ok = new <String>Object();
+    Object notOkStart = new<String>Object();
```

Comment:
Please mark these lines with the `// violation` comment.

Reply:
done